mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -273,7 +333,9 @@ def ncr(n, r):
         parallel = Parallel(n_jobs=n_jobs, pre_dispatch=self.pre_dispatch)
         work = enumerate(parallel(delayed(_calc_score)
                                   (self, X_, y, c, groups=groups, **fit_params)
-                                  for c in candidates))
+                                  for c in candidates)
+                                  if not fixed_feature or
+                                  fixed_feature.issubset(set(c)))
```

Comment:
I think we should be able to do:
```
for c in candidates if fixed_features_set.issubset(set(c))
```

(btw, I am not sure if we need to do `self.fixed_features_set.issubset(set(c))` or not)